SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -17,7 +17,7 @@ If you've signed up for the competition, you'll receive the access details for y
 
 The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.
 
-During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions.
+During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions. After the main presentation we'll move into microgames, which are designed to help you et familiar with the simulator and robot. If you'd like a blue-shirt to sit in on this call to support competitors and answer any further questions, please [let us know](mailto:teams@studentrobotics.org). Note that for safeguarding reasons, we may only be in the call whilst you are.
```

Comment:
Oh, I was expecting this to be a lot more dynamic than needing to let us know ahead of time -- more that we'd jump into a call in response to specific requests.

Separately, have we had an OK from @Scarzy that this is something we can do with our current safeguarding setup? I think knowing we can do this is a blocker to telling team-leaders that we will be doing it.

Reply:
I've updated this to have the primarily method be discord, as it's much easier. We lose voice, but it's still low latency, and public (at least to us). It also makes sharing screenshots and files much simpler.

Reply:
> you et familiar
typo

> Separately, have we had an OK from @Scarzy that this is something we can do with our current safeguarding setup? I think knowing we can do this is a blocker to telling team-leaders that we will be doing it.
This is basically no different to the video interviews we were planning to run prior to the last competition, with the added advantage that as it's targetting a whole team and as such 1-to-1's are far less likely to occur. So fundamentally, no, there's no reason why this couldn't occur.
We had said for the video interviews that it required 2x blueshirts, which is harder to manage with the dynamic approach. However, as the odds of 1-to-1 is reduced it may be possible to forego this requirement. I'll review over the next couple of days and produce specific guidance.

> Note that for safeguarding reasons, we may only be in the call whilst you are.
This is a tad vague as to who "you" is.
We could get away with it not being the team leader, so long as other adults were involved. However team leader is vastly preferable, as such I wouldn't advertise that option, just bring it up if anyone specifically asks.